### PR TITLE
OCPBUGS#42398: Add the service account spec and to Upgrade edge docs

### PR DESCRIPTION
**Pull request by michaelryanpeter**
Version(s): 4.17

Issue: [OCPBUGS-42398](https://issues.redhat.com/browse/OCPBUGS-42398)

Link to docs preview:
* https://82515--ocpdocs-pr.netlify.app/openshift-enterprise/latest/extensions/ce/upgrade-edges.html#olmv1-about-target-versions_upgrade-edges
* https://82515--ocpdocs-pr.netlify.app/openshift-enterprise/latest/extensions/ce/upgrade-edges.html#olmv1-forcing-an-update-or-rollback_upgrade-edges
* https://82515--ocpdocs-pr.netlify.app/openshift-enterprise/latest/extensions/arch/operator-controller.html#olmv1-about-target-versions_operator-controller

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by michaelryanpeter** [comment]
@jianzhangbjz or @Xia-Zhao-rh PTAL

**Comment by Xia-Zhao-rh** [comment]
LGTM

File: modules/olmv1-about-target-versions.adoc [hunk 1]
```diff
@@ -29,6 +29,8 @@ metadata:
 spec:
   packageName: openshift-pipelines-operator-rh
   installNamespace: <namespace_name>
+  serviceAccount:
+    name: <service_account>
```
Comment (xenolinux) [inline modules/olmv1-about-target-versions.adoc:33]: Since this var is being added now, does it need a call out and description?

**Review by xenolinux — COMMENTED** [review]
One question; otherwise LGTM!

File: modules/olmv1-about-target-versions.adoc (hunk 1 above)
Comment (michaelryanpeter) [reply modules/olmv1-about-target-versions.adoc:33]: I am going to add an xref to the supporting documentation on service accounts once that content merges. I debated adding a call out here for the sake of completeness. When I tried it, it actually distracted from the point of the module, which is focused on upgrade edges.

Comment (michaelryanpeter) [reply modules/olmv1-about-target-versions.adoc:33]: Anyhow, that is just to say that I think your suggestion is a good idea, but it didn't work for this content in the end. Thanks, @xenolinux!

**Pull request merged**